Storage/docu/docu_3_code.cc

```diff
@@ -415,6 +415,95 @@ TEST(StorageDocumentation, RESTifiedStorageExample) {
 
 }
 
+TEST(StorageDocumentation, IfUnmodifiedSince) {
+  current::time::ResetToZero();
+
+  using namespace current::storage::rest;
+  using namespace storage_docu;
+  using TestStorage = StorageOfClients<SherlockInMemoryStreamPersister>;
+
+  TestStorage storage;
+
+  const auto basic_rest = RESTfulStorage<TestStorage>(
+      storage, FLAGS_client_storage_test_port, "/api_basic", "http://example.current.ai/api_basic");
+  const auto rest = RESTfulStorage<TestStorage, current::storage::rest::Hypermedia>(
+      storage, FLAGS_client_storage_test_port, "/api", "http://example.current.ai/api");
+
+  const auto base_url = current::strings::Printf("http://localhost:%d", FLAGS_client_storage_test_port);
+
+  // POST one record.
+  const auto publish_time = std::chrono::microseconds(1467363600000000);  // `Fri, 01 Jul 2016 09:00:00 GMT`.
```

Comment:
Nit: Instead of a comment, an `ASSERT_EQ` parsing the string and comparing it to this `const auto publish_time`.